src/encoded/schemas/target.json

```diff
@@ -38,17 +38,39 @@
             "type": "string",
             "linkTo": "Organism"
         },
+        "targeted_genes": {
+            "title": "Targeted genes",
+            "description": "List of gene(s) that are targeted by corrsponding assay or antibody.",
+            "comment": "See gene.json for available identifiers and details.",
+            "type": "array",
+            "uniqueItems": true,
+            "default": [],
+            "items": {
+                "title": "Targeted Gene",
+                "description": "A gene that are targeted by corrsponding assay or antibody.",
+                "comment": "See gene.json for available identifiers.",
+                "type": "string",
+                "linkTo": "Gene"
+            }
+        },
         "gene_name": {
             "title": "Gene name",
             "description": "HGNC or MGI identifier for the target.",
             "comment": "Submit only the identifier (e.g. HMFN0395 or 22809).",
             "type": "string",
             "format": "gene_name"
         },
+        "modification": {
+            "title": "Modification to the corresponding gene.",
+            "description": "A textual description of modification(s) made to the corresponding gene in this targets (e.g. epitope tag(s), post-translational modifications)",
+            "comment": "This property will be used to calculate the proper label of the target. See computed_label property for details.",
+            "type": "string",
+            "default": ""
```

Comment:
We shouldn't do defaults or blank defaults if this is not a required property. I think we should think about making this import_items if It stays a string for now or make it a set of restricted enums matching the enums for GeneticModification.introduced_tags

Reply:
yes I think that is correct; we want it to default to null (no property exists), not "".